ChiModules/KEigenvalueSolver/IterativeOperations/set_k_source.cc

```diff
@@ -121,14 +115,14 @@ void KEigenvalue::Solver::SetKSource(LBSGroupset& groupset,
           q_mom[g] += fission_g;
 
           // ----- Contribute precursors
-          precursor_g = 0.0;
+          double precursor_g = 0.0;
           if ((ell == 0) and (options.use_precursors))
           {
             if ((apply_mat_src) and (xs->J > 0))
```

Comment:
I guess J is the # of precursors. Could we use a better name for that variable than just J?

Reply:
I agree. This however would require a number of changes not in the scope of this PR. Would you be ok with me creating an issue for it to the code author (Zach Hardy)?